client/blocks/inline-help/style.scss

```diff
@@ -0,0 +1,257 @@
+.sidebar .sidebar__footer .button,
+.editor-sidebar .sidebar__footer .button {
+	&.inline-help {
+		border-radius: 0;
+		border-left: 1px solid #c8d7e1;
+		flex: 0 1 40px;
+		margin-right: 0;
+		margin-left: auto;
+		outline: 0;
+
+		@include breakpoint( "<660px" ) {
+			flex: 0 1 56px;
+		}
+
+		.gridicon {
+			margin: 0 auto;
+			float: none;
+		}
+	}
+
+	&.is-active {
+		background: $blue-medium;
+
+		.gridicon {
+			fill: $white;
+		}
+	}
+}
+
+.editor-sidebar .sidebar__footer .button {
+	&.inline-help {
+		flex: 0 1 60px;
+	}
+}
+
+.layout.is-section-help .sidebar .inline-help {
+	background: $gray-text-min;
+	
+	.gridicon {
+		fill: $white;
+	}
+}
+
+.popover.inline-help__popover {
+	@include breakpoint( "<660px" ) {
+		position: fixed;
+			top: 47px !important;
+			right: 0 !important;
+			bottom: 0 !important;
+			left: 0 !important;
+		background: rgba( $gray-light, 0.8 );
+
+		.popover__arrow {
+			display: none;
+		}
+
+		.popover__inner {
+			top: auto !important;
+			right: auto !important;
+			bottom: auto !important;
+			left: auto !important;
+			margin: 16px;
+		}
+	}
+
+	@include breakpoint( ">660px" ) {
+		width: 320px;
+
+		position: fixed;
+			top: auto !important;
+			bottom: 35px !important;
+	}
+
+	&.is-top .popover__arrow:before,
+	&.is-top-left .popover__arrow:before,
+	&.is-top-right .popover__arrow:before {
+		border-top: 10px solid #f3f6f8;
+	}
+
+	.button.inline-help__button {
+		text-align: left;
+		font-weight: 400;
+		width: 100%;
+		background: $gray-light;
+		border-top: 1px solid lighten( $gray, 20% );
+		padding: 16px 16px 16px 48px;
+		font-size: 16px;
+		line-height: 1;
+		position: relative;
+		border-radius: 0 0 4px 4px;
+
+		@include breakpoint( ">660px" ) {
+			font-size: 14px;
+			padding: 12px 16px 12px 48px;
+		}
+
+		.gridicon {
+			position: absolute;
+				top: 13px;
+				left: 13px;
+
+			@include breakpoint( ">660px" ) {
+				top: 8px;
+			}
+		}
+	}
+}
+
+.inline-help__heading {
+	.search-card {
+		border-radius: 4px 4px 0 0;
+		margin: 0;
+
+		.search {
+			border-radius: 4px 4px 0 0;
+			height: auto;
+
+			&.is-open.has-focus {
+				box-shadow: 0 0 0 1px $blue-wordpress,
+					0 0 0 6px $blue-light;
+
+				@include breakpoint( ">660px" ) {
+					box-shadow: 0 0 0 1px $blue-wordpress,
+						0 0 0 4px $blue-light;
+				}
+			}
+
+			.search__input {
+				font-size: 18px;
```

Comment:
Why do you have to touch font-sizes? There's a `compact` version of search if needed.